### ISSUE TITLE
[Bug Report] SRM parsing error: Error updating controller templates

### ISSUE THREAD

**Issue by rainlinx** [dragoonDorise/EmuDeck#1217]
### Prerequisite Steps

- [X] I have searched existing issues
- [X] This issue is not a duplicate of an existing one
- [X] I have checked the [EmuDeck Wiki](https://emudeck.github.io/)
- [X] I have read the [Troubleshooting Page](https://emudeck.github.io/troubleshooting/steamos/) on the EmuDeck Wiki

### What distro are you running?

SteamOS

### Which SteamOS Branch are you using?

Stable

### What happened?

I encountered a problem when using the parser provided by emudeck, here is the log:
"Error updating controller templates. See event log for details.
Controller error: Cannot read properties of undefined ( reading '-965578072' )"
I have tried reinstalling srm, but the problem still exists. 

### Relevant log output

```shell
Error updating controller templates. See event log for details.
Controller error: Cannot read properties of undefined ( reading '-965578072' )
```

**Comment by cbartondock** [comment]
This will be fixed in the next SRM release.

**Comment by rainlinx** [comment]
I have updated to version 2.4.27 and still get error.

**Comment by cbartondock** [comment]
Fixed in 2.4.28
